docs/reference/gcp/queues.mdx

```diff
@@ -0,0 +1,15 @@
+Nitric queues are deployed using [pubsub](https://cloud.google.com/pubsub) topics with a single pull subscription when deploying to GCP.
+
+## GCP Resources
+
+The following resources are created when deploying queues to GCP:
+
+- Pubsub topic
+- Pubsub topic subscription (pull)
+
+## Deployment
+
+During deployment Nitric's CLI builds your Queues will be created as follows:
```

Comment:
Again, reads confusingly. Should it be similar to the schedules deployment?

During deployment, your queues will be provisioned as follows: